app/controllers/home_controller.rb

```diff
@@ -1,41 +1,31 @@
 class HomeController < ApplicationController
   def index
-    # Get the top 10 posts on the front page in the format of {title: upvotes}
-    @subscriber = Subscriber.hashify(Subscriber.top_ten)
+    @subscribers = Subscriber.hashify(Subscriber.top_ten)
     # Timeframes allowed in current dropdown list
-    @timeframe = { '5 minutes' => 5,
+    @timeframe = { '5 minutes'  => 5,
                    '30 minutes' => 30,
-                   '1 hour' => 60,
-                   '6 hours' => 360,
-                   '12 hours' => 720,
-                   '24 hours' => 1440 }
+                   '1 hour'     => 60,
+                   '6 hours'    => 360,
+                   '12 hours'   => 720,
+                   '24 hours'   => 1440 }
   end
 
   def timeframe
-    # Get the top 10 posts on the front page in the
-    # format of {title: upvotes} for this timeframe
-    @subscriber = Subscriber.title_score_hash_timeframe(time_params) rescue nil
-    # if we dont get at least 10 results back, we don't have
-    # proper DB data, kick back an alert message and redirect
-    return render partial: 'home/nodata.js.erb' unless @subscriber.present?
-    # Render the new results on the page
-    render partial: 'home/chartdata.js.erb'
+    @subscribers = Subscriber.title_score_hash_timeframe(time_params)
+    if @subscribers
+      render partial: 'chartdata.js.erb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.